src/main/java/com/google/api/codegen/config/RetryCodesConfig.java

```diff
@@ -203,10 +212,30 @@ private void populateRetryCodesDefinitionWithProtoFile(
 
     // Unite all HTTP GET methods that have no additional retry codes under one retry code name to
     // reduce duplication.
-    String httpGetRetryName = symbolTable.getNewSymbol(HTTP_RETRY_CODE_DEF_NAME);
+    String httpGetRetryName;
+    if (new HashSet<>(RETRY_CODES_FOR_HTTP_GET)
```

Comment:
Nit: personal preference here, but I think breaking this into multiple statements (i.e. declare hashsets, then compare them) is quite a lot clearer.

Reply:
done